crates/forge/tests/integration/runtime.rs

```diff
@@ -24,7 +24,39 @@ fn missing_cheatcode_error() {
     assert_case_output_contains(
         &result,
         "missing_cheatcode_error",
-        "Cheatcode `not_existing123` is not supported in this runtime",
+        indoc!(
+            r"
+            Cheatcode `not_existing123` is not supported in this runtime
+            Check if used snforge_std is compatible with used snforge binary
```

Comment:
It might also be the case for things inside `sncast`'s script so it's not the only possible outcome here. We should probably have a broader explanation